src/container/srv_target.c

```diff
@@ -1262,6 +1262,8 @@ cont_child_destroy_one(void *vin)
 	}
 
 	if (cont->sc_destroying) {
+		D_ERROR(DF_CONT ": Container is already being destroyed\n",
```

Comment:
this doesn't sound like should be logged as an error but more like a debug?

Reply:
Indeed, let me fix this...